src/modules/Users.pm

```diff
@@ -4466,12 +4493,14 @@ sub Write {
 		$chown_home	        = 1 if (!defined $chown_home);
 		my $skel	        = $useradd_defaults{"skel"};
 
+		y2milestone ("User = ", Dumper(\%user));
```

Comment:
BTW what does this `%user` data actually contain? We need to be careful, if that contains the password we should not write it into the log.